instance_control/config.py

```diff
@@ -0,0 +1,48 @@
+import json
+import re
+
+import logging
+import requests
+
+_LOG = logging.getLogger('bubuku.cluster.config')
+KAFKA_LOGS_EBS = 'kafka-logs-ebs'
+
+
+def read_cluster_config(cluster_name: str, cluster_config_path: str):
```

Comment:
do you think it is less error prone? or what is the difference? we have only 2 configs for now